runners/spark/src/test/java/org/apache/beam/runners/spark/translation/StorageLevelTest.java

```diff
@@ -17,29 +17,29 @@
  */
 package org.apache.beam.runners.spark.translation;
 
-import org.apache.beam.runners.spark.PipelineRule;
+import org.apache.beam.runners.spark.SparkPipelineOptions;
+import org.apache.beam.runners.spark.SparkRunner;
 import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
 import org.apache.beam.sdk.testing.PAssert;
 import org.apache.beam.sdk.transforms.Count;
 import org.apache.beam.sdk.transforms.Create;
 import org.apache.beam.sdk.values.PCollection;
-import org.junit.Rule;
 import org.junit.Test;
 
 /**
  * Test the RDD storage level defined by user.
  */
 public class StorageLevelTest {
```

Comment:
I want to figure this one out.. it is weird and might point to a larger/different problem.
I tested the "PipelineRule" version of this test with removing the caching in `EvaluationContext#computeOutputs` and it passed.. but broke other tests.
Let me get back to you on this.